.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.12.2
+current_version = 0.12.3
```

Comment:
Can we revert the bumpversion call please? I'll handle the version bump when a new version is released.